interview-questions.md

```diff
@@ -12,46 +12,54 @@ Your answer:
 
 Researched answer:
 
-1. What is git? What is the difference between git and Github?
+1. What is git? What is the difference between git and Github? 
 
-Your answer:
+Your answer: git is a method of version control.  Utilizing the commands of git add and git commit, you are able to track the ways in which you modify your code.  GitHub is a cloud based repository in which your code is uploaded and made accessible to others.
 
-Researched answer:
+Researched answer: While both git and GitHub are ways to maintain version control, git functions on the local machine, while GitHub exists in the cloud (remote storage)
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: The JavaScript operators that will return a Boolean Value are any of the conditional operators.  These include < (less than), <= (less than or equal to), > (greater than), >= (greater than or equall to), != (not equal to), == or === (loose or strict interpretation of a value),
+&& (checking to see if two conditions are both true), or || (checking to see if one of two given conditions is true).
 
-Researched answer:
+Researched answer: The operators I listed above are correct, in that they will return a Boolean value.  I also found that there were additional conditional operators I had missed, such as !== (not equal type or equal value) and ? (ternary operator). 
```

Comment:
Like that you are using the research to verify your answers. This question is seeking the three type of operators that produce a Boolean value. You have given great examples. Now list the examples under their appropriate categories.